### DISCUSSION TITLE
FluentDatePicker styling

### DISCUSSION THREAD

**Discussion by Nehakrg** [microsoft/fluentui-blazor#1756] [Q&A]
![image](https://github.com/microsoft/fluentui-blazor/assets/25210450/d0d7734f-3520-49f3-ab89-7b1769936ad2)

**Comment by vnbaaij** [comment]
Next time, please use normal text instead of an image...

1. Masking isnot supported 
2. Add `DisabledSelectable="false"` to get rid of the strikethrough
3. Only the days from the current mont are selectable. Besides using the arrows, a user can click on month to quickly select any mont of the year or click on the yesr to select from a group of 12 years. See also https://www.fluentui-blazor.net/DateTime#defaultcalendar
